src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  function onChangeTabHandler(event, index) {
+    event.preventDefault();
+    onTabSelected(tabs[index]);
+  }
```

Comment:
```suggestion
  function onChangeTabHandler(tab) {
    event.preventDefault();
    onTabSelected(tab);
  }
```

Reply:
can i use 'event' to prevent default behavior of 'a' element without passing it into a function?

Reply:
i see. thank you.

Reply:
but there is another problem after moving to <li> - seems like eslint issue. 
"Visible, non-interactive elements with click handlers must have at least one keyboard listener."

Reply:
so move it back and remove preventDefault just